### PR TITLE
Add link to Sprig library

### PR DESCRIPTION
**Pull request by marksou**

### CODE REVIEW

File: tyk-docs/content/advanced-configuration/transform-traffic/request-body.md [hunk 1]
```diff
@@ -233,3 +233,7 @@ It is possible to work with inbound form data by making use of the Context Varia
 You do this by using the `._tyk_context.` namespace, unlike the context exposed to the URL rewriter and header injector, the body transform can fully iterate through list indices, so for example calling `{{ index ._tyk_context.request_data.variablename 0 }}` in a template will expose the first entry in the `request_data.variablename` key/value array.
 
 The `request_data` section is populated if the inbound request contained any query data or form data, it will be available in this object as a `key:[]value` map.
+
+### Go Template Functions
+
+For increasing the functions available for Go templates, look at the [Sprig Library](http://masterminds.github.io/sprig/) which offers over 70 Go template functions.
```
Comment (janeves) [inline tyk-docs/content/advanced-configuration/transform-traffic/request-body.md]: For increasing the functions available to the built in Go templating, we have bundled the Sprig library which provides over 70 additional functions for transformations.

File: tyk-docs/content/advanced-configuration/transform-traffic/response-body.md [hunk 2]
```diff
@@ -78,3 +78,7 @@ A middleware called `header_transform`, added in v2.1, ensures headers such as `
 In this configuration, you set the `headers` to target and the `target host` to replace the values with. In the above example, the `Link` and `Location` headers will be modified from the server-generated response, with the protocol, domain and port of the value set in `target_host`.
 
 (This is not supported in the Dashboard yet.)
+
+### Go Template Functions
+
+For increasing the functions available for Go templates, look at the [Sprig Library](http://masterminds.github.io/sprig/) which offers over 70 Go template functions.
```
Comment (janeves) [inline tyk-docs/content/advanced-configuration/transform-traffic/response-body.md]: For increasing the functions available to the built in Go templating, we have bundled the Sprig library which provides over 70 additional functions for transformations.

**Review by janeves — CHANGES_REQUESTED** [review]

File: tyk-docs/content/advanced-configuration/transform-traffic/request-body.md (hunk 1 above)
Comment (marksou) [reply tyk-docs/content/advanced-configuration/transform-traffic/request-body.md]: Done

File: tyk-docs/content/advanced-configuration/transform-traffic/response-body.md (hunk 2 above)
Comment (marksou) [reply tyk-docs/content/advanced-configuration/transform-traffic/response-body.md]: Done

**Pull request merged**